client/app/styles/rjsf/TitleFieldTemplate.tsx

```diff
@@ -0,0 +1,7 @@
+const TitleFieldTemplate = () => {
+  // RJSF isn't giving us access to 'ui:options' or 'ui:classNames' for customization
+  // so we're just going to return an empty fragment and handle the title in the field template
+  return <></>;
+};
+
+export default TitleFieldTemplate;
```

Comment:
@BCerki what are your thoughts on this?

Reply:
Thanks for taking another look. That `ui:options` is specific to the `multiple_operator_array` field though. I am not sure why they are sanitizing the main options even though they are supported in other templates.

Sounds good though, I still think this fix is better than hiding it with CSS selectors so I will merge it in even though it's imperfect.